sr/main.go

```diff
@@ -73,11 +73,43 @@ func main() {
 			Usage:  "sr schema 7878",
 			Action: schema,
 		},
+		{
+			Name:   "config",
+			Usage:  "sr config [subject]",
+			Action: config,
+		},
+		{
+			Name:   "set-config",
+			Usage:  "sr set-config foo FULL",
```

Comment:
`[]` does well for communicating optional so perhaps just 'subject' then?  Its fine how it is though.